code4lib/2020.code4lib.org
# add Schedule

closes #103 

to test

- Schedule main menu link should take you to /schedule/index.html now
- check each day's schedule against [the spreadsheet](https://docs.google.com/spreadsheets/d/1bXLUTgBNQVJBcLssy0ukQoGhkPuA8SUdrDlteHNyYFU/edit?ts=5df2bd4a#gid=164871641)
- double-check several talk pages to ensure they show the right info
